src/piece.cpp

```diff
@@ -110,6 +110,14 @@ namespace {
       p->stepsCapture = {2 * SOUTH_WEST, 2 * SOUTH_EAST, 2 * NORTH_WEST, 2 * NORTH_EAST};
       return p;
   }
+  PieceInfo* dabbaba_piece() {
+      PieceInfo* p = new PieceInfo();
+      p->name = "dabbaba";
```

Comment:
I assume this piece name is just used so that `variants.ini` (and maybe elsewhere) can refer to the piece name, so this just needs to match my `variants.ini` change (which it does)